src/passes/externalInputChecker/intBoundChecker.ts

```diff
@@ -0,0 +1,86 @@
+import { AST } from '../../ast/ast';
+import {
+  FunctionDefinition,
+  FunctionVisibility,
+  Statement,
+  ExpressionStatement,
+  ElementaryTypeName,
+  Identifier,
+  IntType,
+  VariableDeclaration,
+  FunctionCall,
+} from 'solc-typed-ast';
+import { ASTMapper } from '../../ast/mapper';
+import { createCairoFunctionStub, createCallToFunction } from '../../utils/functionStubbing';
+import { typeNameFromTypeNode } from '../../utils/utils';
+import assert = require('assert');
+
+export class IntBoundChecker extends ASTMapper {
+  visitFunctionDefinition(node: FunctionDefinition, ast: AST): void {
+    if (FunctionVisibility.External === node.visibility) {
+      node.vParameters.vParameters.forEach((parameter) => {
+        if (
+          (parameter.typeString.slice(0, 4) === 'uint' ||
```

Comment:
This is minor and not required, but it might be nicer to do getNodeType on the parameter, check if it's an IntType, then use the width stored in the IntType rather than manually parsing typeStrings

Reply:
Replaced it.